### PR TITLE
feat : Added P1 - Ratings component

### PR DESCRIPTION
**Pull request by TusharJade**
###   Added P1 - Ratings component
Screenshot (47)

### CODE REVIEW

File: Components/Ratings/ratings.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.filled{
+    color: grey;
```
Comment (Avin008) [inline Components/Ratings/ratings.css]: use color variable

File: Components/Ratings/ratings.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        <p class="des-para">This is 1 star rating component which you can use in your ecommerce</p>
+        <div class="rating-outdiv">
+            <i class="fas fa-star filled"></i>
+        </div>
+
+
+       
+
+       
+
```
Comment (Avin008) [inline Components/Ratings/ratings.html]: remove black space

**Review by Avin008 — APPROVED** [review]
looks good to merge

File: Components/Ratings/ratings.css (hunk 1 above)
Comment (TusharJade) [reply Components/Ratings/ratings.css]: ok will fix it

File: Components/Ratings/ratings.html (hunk 2 above)
Comment (TusharJade) [reply Components/Ratings/ratings.html]: yup

**Review by SumitSaundadkar — APPROVED** [review]
Use prettier code as much as possible, avoid spacing,think on class name ...looks good to me..good work..keep it up

**Comment by TusharJade** [comment]
> Use prettier code as much as possible, avoid spacing,think on class name ...looks good to me..good work..keep it up

Yup I used beautify most of the the time. Thanks for feedback 🙌

**Pull request merged**